makefile-docker.defs

```diff
@@ -3,7 +3,14 @@
 # ----------------------------------
 # starting, stopping, migrating DB
 # ----------------------------------
-start:
+
+create_mock_vespa_certs:
```

Comment:
Create the local vespa certs only if they don't exist upon `make start`

Reply:
Nice addition!